wagtail/contrib/forms/views.py

```diff
@@ -221,25 +236,6 @@ def get_ordering(self):
         ordering = self.get_validated_ordering()
         return [values[0] + name for name, values in ordering.items()]
 
-    def get_filtering(self):
-        """Return filering as a dict for submissions queryset"""
-        self.select_date_form = SelectDateForm(self.request.GET)
-        result = {}
-        if self.select_date_form.is_valid():
-            date_from = self.select_date_form.cleaned_data.get("date_from")
-            date_to = self.select_date_form.cleaned_data.get("date_to")
```

Comment:
Using `date` as the filter name (with `field_name="submit_time"`), along with changing the default `DateRangePickerWidget.suffixes` allowed us to have the exact same parameters for these filters (and saves us from updating the form submission's tests). Not the best reasons to do those things, but I think it's still worth renaming the suffixes for reasons I mentioned in the other comment.